scripts/parachain-launch/statemine.yml

```diff
@@ -0,0 +1,40 @@
+relaychain:
+  image: parity/polkadot:v0.9.29
+  chain: rococo-local
+  runtimeGenesisConfig:
+    configuration:
+      config:
+        validation_upgrade_cooldown: 10
+        validation_upgrade_delay: 10
+  env:
+    RUST_LOG: parachain::candidate-backing=trace,parachain::candidate-selection=trace,parachain::pvf=debug,parachain::provisioner=trace
+  flags:
+    - --rpc-methods=unsafe
+    - --wasm-execution=compiled
+    - --execution=wasm
+  nodes:
+    - name: alice
+    - name: bob
+
+parachains:
+- image: docker.io/parity/polkadot-parachain:latest
+  chain:
+    base: statemine-local
+    collators:
+      - alice
+    sudo: alice
+  id: 2000
+  parachain: true
+  flags:
+    - --rpc-methods=unsafe
+    - --force-authoring
+    - --wasm-execution=compiled
+    - --execution=wasm
+  relaychainFlags:
+    - --wasm-execution=compiled
+    - --execution=wasm
+  env:
+    RUST_LOG: sc_basic_authorship=trace,cumulus-consensus=trace,cumulus-collator=trace,collator_protocol=trace,collation_generation=trace,aura=debug
```

Comment:
Do we need them?

Reply:
We dont need them, but it could help if you are trying to debug. 

The parachain-launch template examples include them. 

Would you like them removed?

Reply:
Yes, I would like to have them removed.

Reply:
Removed